README.md

```diff
@@ -99,24 +99,29 @@ Fedora, CentOS, RHEL, and related distributions:
 yum install -y \
   btrfs-progs-devel \
   device-mapper-devel \
+  git \
   glib2-devel \
   glibc-devel \
   glibc-static \
+  go \
   gpgme-devel \
   libassuan-devel \
   libgpg-error-devel \
   libseccomp-devel \
   libselinux-devel \
   ostree-devel \
   pkgconfig \
-  runc
+  runc \
+  skopeo-containers
```

Comment:
Should we deferentiate between build dependencies and run dependencies.  I would say runc and skopeo-containers are runtime dependencies.

Reply:
runc and skopeo-containers are definitely both runtime, you can build with out them (I think, didn't try without runc).   I'm ok with blurring the lines here or changing the title to "Build and Run Dependencies" if you prefer.  My main thing was that on a brand new install, I needed these packages in place to get "kpod pull" working.

Reply:
Yes lets change the heading, for that.

Reply:
Fixed.